### DISCUSSION TITLE
Error while setting up locally on MacOS Sonoma

### DISCUSSION THREAD

**Discussion by pranavkonidena** [oppia/oppia-android#5271] [Q&A (Installation)] [answered]
### Operating System

MacOS Sonoma

### Android Studio Version

Android Studio Bumblebee | 2021.1.1 Patch 3 

### RAM size

16

### Free disk space

200 GB

### Which step are you stuck on?

I got an error while trying to set up the project locally following all the steps mentioned. Finally when I try to run the app from android studio, i get the following error. I think it might have something to do with the rosetta terminal because I saw we can't get an instance of rosetta terminal anymore. (Although I did see that some files were downloaded upon executing the setup script)

### Error Log

```bash
Execution failed for task ':model:generateProto'.
> Could not resolve all files for configuration ':model:protobufToolsLocator_protoc'.
   > Could not find protoc-3.8.0-osx-aarch_64.exe (com.google.protobuf:protoc:3.8.0).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.8.0/protoc-3.8.0-osx-aarch_64.exe

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

### Approaches already used to resolve the issue

I have searched online and went through stack overflow suggested solutions but to no avail.

**Comment by MohitGupta121** [comment, marked as answer]
@pranavkonidena you can try solution in this [disscussion](https://github.com/oppia/oppia-android/discussions/4526) this might helps. Thanks!

**Reply by pranavkonidena** [reply]
Thank you very much!
